code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -41,6 +42,13 @@
 	if(H.dna.features["moth_wings"] != "Burnt Off" && H.bodytemperature >= 800 && H.fire_stacks > 0) //do not go into the extremely hot light. you will not survive
 		to_chat(H, "<span class='danger'>Your precious wings burn to a crisp!</span>")
 		H.dna.features["moth_wings"] = "Burnt Off"
+		if(flying_species) //This is all exclusive to if the person has the effects of a potion of flight
```

Comment:
can you remove all of these moth changes except the language one? i've handled them in gh#2143